### PR TITLE
optional year for multiple destinations

### PR DESCRIPTION
**Pull request by mciparelli**

### CODE REVIEW

File: utils/parser.js [hunk 1]
```diff
@@ -89,12 +89,25 @@ const belongsToCity = (airport, city) => {
   }
 };
 
+function findMonthAndYearFromText(text) {
```
[the remark this answers is not in the document]
Comment (mciparelli) [reply utils/parser.js:92]: Yeah I can change it to use arrow functions. The _ prefix is there to indicate that it's actually an unused variable, but yeah I guess I can remove it from the variables if that's a preference

**Comment by mciparelli** [comment]
Care to give it another quick look @juanidambrosio ?

File: config/constants.js [hunk 2]
```diff
@@ -5,8 +5,17 @@ const badResponse = "El formato es inválido.";
 const dailyTweet =
   "Te ayudamos a encontrar las 10 mejores ofertas en millas! Respondé este tweet con el formato [ORIGEN] [DESTINO] [AÑO-MES]. Ejemplo EZE MAD 2023-05.";
 
+const currentDate = new Date();
+
+const currentYear = currentDate.getFullYear();
+const currentMonth = currentDate.getMonth() + 1;
+const padMonth = someMonth => {
```
Comment (juanidambrosio) [inline config/constants.js:12]: Could we call to this function from the utils folder? I thinks its better to keep only constants in here

File: telegram/search.js [hunk 3]
```diff
@@ -74,7 +74,7 @@ const searchCityQuery = async (msg, isAlert) => {
           generateFlightOutput(current) +
           "\n"
         ),
-    payload.origin + " " + payload.destination + "\n"
+    payload.origin + " " + payload.destination + " " + payload.departureDate + "\n"
```
Comment (juanidambrosio) [inline telegram/search.js:77]: Why is this addition at the results title? The dates are detailed in each flight row

**Comment by juanidambrosio** [comment]
Reviewed, with that minor change we can rebase and merge

**Comment by mciparelli** [comment]
> Reviewed, with that minor change we can rebase and merge

k, updated.

**Pull request merged**